integration-test-common/build.gradle.kts

```diff
@@ -54,25 +62,39 @@ dependencies {
     exclude("org.elasticsearch")
     exclude("org.elasticsearch.client")
     exclude("org.elasticsearch.plugin")
+    exclude("com.amazonaws", "aws-java-sdk-bundle")
   }
+  testImplementation(libs.junit.jupiter.params)
+  testImplementation(libs.junit.jupiter.api)
+  testImplementation(libs.junit.jupiter.engine)
+  testImplementation(libs.bundles.jersey)
+}
 
-  testImplementation(platform("org.junit:junit-bom:5.9.1"))
-  testImplementation("org.junit.jupiter:junit-jupiter")
+val testShadowJar by tasks.registering(ShadowJar::class) {
```

Comment:
after using `ShadowJar`, the integration test common jar will include almost all dependences, is it expected?

Reply:
The `integration-test-comman` ShadowJar main use to running `MiniGravitino` service.
We can expend a single `MiniGravitino` module in the Gravitino project in the future.
@mchades What do you think?

Reply:
Yeah, just like mini HMS. It can be imported and setup directly by other programs without introducing additional dependencies.